WordPress/Sniffs/VIP/SessionVariableUsageSniff.php

```diff
@@ -24,9 +24,32 @@
  *                 which it didn't use.
  * @since   0.12.0 This class now extends WordPress_Sniff.
  * @since   0.13.0 Class name changed: this class is now namespaced.
+ *
+ * @deprecated 1.0.0  This sniff has been deprecated.
+ *                    This file remains for now to prevent BC breaks.
  */
 class SessionVariableUsageSniff extends Sniff {
 
+	/**
+	 * If true, an error will be thrown; otherwise a warning.
+	 *
+	 * @var bool
+	 */
+	public $error = true;
+
+	/**
+	 * Keep track of whether the warnings have been thrown to prevent
+	 * the messages being thrown for every token triggering the sniff.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var array
+	 */
+	private $thrown = array(
+		'DeprecatedSniff'                 => false,
+		'FoundPropertyForDeprecatedSniff' => false,
```

Comment:
This sniff does not have any public properties which can be changed, so the `FoundPropertyForDeprecatedSniff` entry can be removed.